MailCoreUI/Utils/FloatingPanelHelper.swift

```diff
@@ -92,17 +98,34 @@ public struct SelfSizingPanelBackportViewModifier: ViewModifier {
         return topPadding + titleSpacing + UIFont.preferredFont(forTextStyle: .headline).pointSize
     }
 
+    private var shouldShowCloseButton: Bool {
+        return platformDetector.isMac || UIDevice.current.orientation.isLandscape
+    }
+
+    private var shouldShowZStack: Bool {
```

Comment:
```suggestion
    private var shouldShowHeader: Bool {
```
We will prefer identify the views by what their represents